### PR TITLE
Add staging command, and make sudo set the HOME dir

### PR DESCRIPTION
**Pull request by pete-woods**

### CODE REVIEW

File: refresh-bits [hunk 1]
```diff
@@ -132,7 +132,12 @@ while [ "$1" != '' ]; do
             ;;
         run-snapd)
             echo "Running snapd (use sudo snap to talk to it)"
-            ssh "$SNAPPY_HOST" sudo /lib/systemd/systemd-activate -l /run/snapd.socket ./snapd
+            ssh "$SNAPPY_HOST" sudo -H /lib/systemd/systemd-activate -l /run/snapd.socket ./snapd
+            shift
+            ;;
+        run-snapd-staging)
```
Comment (zyga) [inline refresh-bits:139]: Can you document that in the help please?

**Comment by zyga** [comment]
Ping? Can you change the help message to include this so that we can land this? :-)

**Comment by pete-woods** [comment]
Thanks for the reminder :)

Have updated the PR now.

**Comment by zyga** [comment]
+1

I'll probably change that to run-snapd --staging later but this is not a priority for me now :)

**Pull request merged**